using_images/other_images/jenkins.adoc

```diff
@@ -106,6 +106,65 @@ To use a persistent volume store, the cluster administrator must define a link:.
 
 Once selected, you must link:../../dev_guide/templates.html#creating-resources-from-a-template[instantiate] the template to be able to use Jenkins.
 
+== Using Jenkins as an Source-To-Image builder
+
+If you want to customize the official OpenShift Jenkins image, you can use the
+Docker layering or you can use the image as a Source-To-Image builder.
+You can use Source-To-Image to copy your custom Jenkins Jobs definitions, additional
```

Comment:
should link source-to-image here.  look at what we do in our other s2i image documentation for reference:
https://docs.openshift.org/latest/using_images/s2i_images/nodejs.html on how to talk about s2i and what to link to.